tutorial/README.md

```diff
@@ -0,0 +1,476 @@
+
+Click on the `New service` button. Give it a name. For sake of simplicity, we'll make it the same for all days of the week, with a validity period of one year.
+
+![Lesson 7: new service](images/lesson7_newService.png)
+
+Save the service by clicking the save ![save button](images/buttonSave.png) button.
+
+### Create new nodes
+
+Before creating the new line, we need to make sure that all stop nodes exist, where we want to build stations. To add new nodes, go to the `Stop nodes` panel, from the left menu.
+
+![Lesson 7: nodes panel](images/lesson7_nodesPanel.png)
+
+Nodes in the city itself are numerous enough, we will add a few new nodes in the suburb. To add a new node, simply click on the map at the desired location. This will draw the node and open the node edit window.
+
+You can give it a name and a code. If codes of existing nodes are sequential, writing a digit should give a few auto-complete suggestions for the next code.
+
+The radius used for routing means that the actual transit stop should be within this radius from the node. For example, a transit node can be between 2 one way streets, one for each direction. Make sure the radius is sufficient to cover the streets by which vehicles could pass. This radius is represented on the map by the blue circle around the node.
+
+![Lesson 7: new node panel](images/lesson7_newNodePanel.png)
+
+Add as many nodes as may be required for your new line.
+
+Once all nodes have been added/updated, the transferrable nodes should be recalculated by clicking on the `update transferrable nodes` ![update transferrable nodes](images/buttonUpdateTransferrableNodes.png) button (step 1 of the [refresh calculation data section](#refresh-calculation-data)).
+
+### Create a new agency
+
+The new line will be under a new agency. To create the new agency, go to the `Agencies and lines` panel ![agencies and lines panel](images/buttonAgencyLinePanel.png).
+
+Click on the `New agency` button and fill the new agency's data. Click on the save ![save button](images/buttonSave.png) button to save the new agency.
+
+![Lesson 7: new Agency](images/lesson7_newAgencyEdit.png)
+
+You will go back to the agencies list. The new agency should be listed there.
+
+![Lesson 7: agency list with new agency](images/lesson7_agencyListWithNewAgency.png)
+
+### Create a new line
+
+To create a new line, expand the agency's lines by clicking on the `Lines` text under the agency and click on the `New line` button. You can also click on the `New line` button at the bottom of the panel, but you'll have to select the agency manually in the new line.
+
+![Lesson 7: new line button](images/lesson7_newLineButton.png)
+
+Fill the line's basic data: number, name, mode, right-of-way, color. The mode is important as it will determine the default routing method when building the path. For example, a bus line will use the road network, with bus data if available, while a metro will use manual (hand-drawn) routing.
```

Comment:
"For example, a bus line will use the road network, with bus data if available, "

What does "with bus data if available" means?

Reply:
je voulais parler de bus_urban, bus_suburban... probablement pas besoin à ce point-ci... ce sera sélectionné automatiquement si c'est disponible et si pas dispo, ça utilise voiture

Reply:
attention, voiture ne peut pas utiliser les voies réservées et les rues pour bus seulement (il y en a un peu partout), bus_urban et bus_suburb peut, et la vitesse et les pénalités de virages sont beaucoup plus réalistes que voiture. Je les garderais

Reply:
bus data means: reserved lanes, BRT roads, bus-only roads, etc.

Reply:
"For example, a bus line will use the road network, with bus data if available, "
Donc plus: 
"For example, a bus line will use the road network, with extra bus specific information like reserved lanes"

Reply:
done